istioctl/pkg/install/verify.go

```diff
@@ -86,7 +86,7 @@ func verifyPostInstall(enableVerbose bool, istioNamespaceFlag string,
 		}
 		switch kind {
 		case "Deployment":
-			deployment := &v1beta1.Deployment{}
+			deployment := &appsv1.Deployment{}
```

Comment:
Will this support both v1beta and appsv1 now? or just appsv1

Reply:
just appsv1